hw/ip/prim/dv/prim_prince/prim_prince_sim_cfg.hjson

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the name of the DUT.
+  name: prim_prince
+
+  // Top level dut name (sv module).
+  dut: prim_prince
+
+  // Top level testbench name (sv module).
+  tb: prim_prince_tb
+
+  // Simulator used to sign off this block
+  tool: vcs
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:prim_prince_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/prim/dv/prim_prince/data/prim_prince_testplan.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/dv/data/common_sim_cfg.hjson"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  overrides: [
+    {
+      name: vcs_cov_hier
+      value: "-cm_hier {proj_root}/hw/ip/prim/dv/prim_prince/data/cover.cfg"
+    }
+  ]
+
+  // List of test specifications.
+  tests: [
+    {
+      name: prim_prince_sanity
```

Comment:
Probably call this `prim_prince_test` if it is a full test in itself.

Reply:
Yep, will do.